src/manifolds/Stiefel.jl

```diff
@@ -619,3 +596,221 @@ function uniform_distribution(M::Stiefel{n,k,ℝ}, p) where {n,k}
 
     return ProjectedPointDistribution(M, d, project!, p)
 end
+
+@doc raw"""
+    vector_transport_direction(::Stiefel, p, X, d, ::DifferentiatedRetraction{CaleyRetraction})
```

Comment:
Shouldn't it "Cayley" instead of "Caley"?

Reply:
Thanks for noticing the typo.